packages/database/src/prisma-api.ts

```diff
@@ -447,6 +454,16 @@ export function getSdk(
         }
       }
 
+      const merchantPurchaseId = v4()
+
+      const merchantPurchase = ctx.prisma.merchantPurchase.create({
+        data: {
+          id: merchantPurchaseId,
+          checkoutSessionId,
+          merchantAccountId,
+        },
+      })
```

Comment:
Create a `MerchantPurchase` anytime a new purchase is made going forward. This SDK function isn't called for $0 coupon redemption purchases, so we'll only record these when there is a Stripe check session to tie it to.